### ISSUE TITLE
Image lists are different when downloading from Rancher and Github Release Page

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#22644]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug, `Rancher v2.3.0-rc1`

**Steps to reproduce (least amount of steps as possible):**
The image lists are different between the file `rancher-windows-images.txt` downloaded from Rancher UI and the file `rancher-windows-images.txt` downloaded from Github release page: https://github.com/rancher/rancher/releases/tag/v2.3.0-rc1

The image lists for Linux also do NOT match each other.

**Comment by rmweir** [comment]
Would this be because the image was using the default system chart environment variable from package/Dockerfile which may have been more up to date than the one set and used in scripts/
package? If so this may be resolved by https://github.com/rancher/rancher/pull/22734

**Comment by mrajashree** [comment]
@cjellick @rmweir This doesn't look like the same issue. The windows and linux images downloaded from rancher UI don't have rancher-agent images. Also the system tools images aren't included in the list from UI download link, whereas they are included while generating the list for the command from package script https://github.com/rancher/rancher/blob/master/pkg/image/export/main.go#L82

**Comment by thxCode** [comment]
@jiaqiluo I think `rancher-windows-images.txt` should only keep the images using on Windows nodes, the following list includes the required images:
```
- from system images
  + rancher/hyperkube
  + rancher/rke-tools
  + rancher/kubelet-pause
  + rancher/rke-tools
  + rancher/rancher-agent
- from system chart images
  + logging:
    - rancher/fluentd
    - rancher/configmap-reload
  + monitoring:
    - rancher/wmi_exporter-package 
```

@mrajashree , I think we could address this issue and https://github.com/rancher/rancher/issues/22463 together in PR https://github.com/rancher/rancher/pull/22785, FYI:
- https://github.com/rancher/rancher/pull/22821 would drop the stale tool images for logging and alerting
- https://github.com/rancher/system-charts/pull/96 adds a new mark to indicate those system chart images using on Windows

**Comment by jiaqiluo** [comment]
The bug fix is validated in Rancher:v2.3.0-rc4

The content of `rancher-windows-images.txt` from the UI and the Github release page are identical. 

the `rancher-linux-images.txt` from the Github release page has `rancher/rancher:v2.3.0-rc4` but the one from the UI does not.

It makes sense because if you can download the file from Rancher UI you should already have the `rancher/rancher` image.
